lc-hax/Scripts/Loader.cs

```diff
@@ -76,6 +77,11 @@ internal class Loader : MonoBehaviour {
         Loader.AddHaxModules<DisconnectMod>();
         Loader.AddHaxModules<ClearVisionMod>();
         Loader.AddHaxModules<InstantInteractMod>();
+
+        // Dont Disable this line, it is used to fill the Helper in case is injected during a full loaded level.
+        Loader.AddHaxModules<RefreshMod>();
```

Comment:
Let's move this to `AddHaxGameObject` instead, then move `RefreshMod` to the `Core` directory and rename it to `Refresh`.